### PR TITLE
fix: Display correct selected item in navigation drawer

### LINKED ISSUE #1383: Incorrect display of selected item in Navigation Drawer
**Describe the bug**
Select Check-In tab in Navigation Drawer then view activity log of any attendee and then press back to land up in Attendee List Fragment. Now, again view the activity log for any attendee and see the navigation drawer. The tab selected will be Dashboard instead of Check-In.

**Expected behavior**
The selcted tab should be Check-In whenever user is in Check-In or it's child fragments.

**Screenshots**
ezgif-6-ce7c6418d905

**Would you like to work on the issue?**

- [ ] Yes
- [x] No

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1383 

Checklist:

- [ ] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 
Now the selected tab will be in Check-In whenever user is in Check-In or it's child fragments.

Screenshots for the change:

ezgif com-video-to-gif 4

### CODE REVIEW

**Comment by iamareebjamal** [comment]
This will fail other things, there was a reason this check was in place

**Comment by ShridharGoel** [comment]
Did changes accordingly.

**Comment by ShridharGoel** [comment]
The Travis CI build is still failing.

**Comment by Masquerade0097** [comment]
@SridharGoel The `com.eventyay.organizer.core.main.FragmentNavigatorTest` is failing.

**Comment by ShridharGoel** [comment]
@Masquerade0097 Fixed.

File: app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java [hunk 1]
```diff
@@ -115,8 +115,6 @@ else if (eventId == -1)
             finish();
         else {
             fragmentNavigator.back();
-            binding.navView.getMenu().findItem(R.id.nav_dashboard).setChecked(true);
```
Comment (sridharjajoo) [inline app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java]: What happens when back button is pressed from Attendees fragment. Does the Dashboard item get highlighted and title of toolbar changes to Dashboard?

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java]: Title of toolbar is changing to Dashboard but Dashboard item is not getting highlighted. I'll fix it.

**Pull request closed without merging**